db/migrate/20160505005253_add_send_count_to_invitations.rb

```diff
@@ -0,0 +1,5 @@
+class AddSendCountToInvitations < ActiveRecord::Migration
+  def change
+    add_column :invitations, :send_count, :integer, null: true, default: 0
```

Comment:
`accepted_at` is probably a good index too; I was thinking of doing `WHERE send_count IS NOT NULL` to ignore all of the heaps of existing invitations we have.